dask_cuda/cli/dask_cuda_worker.py

```diff
@@ -19,154 +19,160 @@
 
 @click.command(context_settings=dict(ignore_unknown_options=True))
 @click.argument("scheduler", type=str, required=False)
-@click.option(
-    "--tls-ca-file",
-    type=pem_file_option_type,
-    default=None,
-    help="CA cert(s) file for TLS (in PEM format)",
-)
-@click.option(
-    "--tls-cert",
-    type=pem_file_option_type,
-    default=None,
-    help="certificate file for TLS (in PEM format)",
-)
-@click.option(
-    "--tls-key",
-    type=pem_file_option_type,
-    default=None,
-    help="private key file for TLS (in PEM format)",
-)
-@click.option("--dashboard-address", type=str, default=":0", help="dashboard address")
-@click.option(
-    "--dashboard/--no-dashboard",
-    "dashboard",
-    default=True,
-    show_default=True,
-    required=False,
-    help="Launch dashboard",
+@click.argument(
+    "preload_argv", nargs=-1, type=click.UNPROCESSED, callback=validate_preload_argv
 )
 @click.option(
     "--host",
     type=str,
     default=None,
-    help="Serving host. Should be an ip address that is"
-    " visible to the scheduler and other workers. "
-    "See --listen-address and --contact-address if you "
-    "need different listen and contact addresses. "
-    "See --interface.",
+    help="""Serving host; should be an IP address visible to the scheduler and other
+    workers""",
 )
 @click.option(
-    "--interface",
-    type=str,
-    default=None,
-    help="The external interface used to connect to the scheduler, usually "
-    "an ethernet interface is used for connection, and not an InfiniBand "
-    "interface (if one is available).",
+    "--nthreads",
+    type=int,
+    default=1,
+    show_default=True,
+    help="Number of threads per process",
 )
-@click.option("--nthreads", type=int, default=1, help="Number of threads per process.")
 @click.option(
     "--name",
     type=str,
     default=None,
-    help="A unique name for this worker like 'worker-1'. "
-    "If used with --nprocs then the process number "
-    "will be appended like name-0, name-1, name-2, ...",
+    help="""A unique name for this worker like ``worker-1``; if used with ``--nprocs``,
```

Comment:
This is just nipticking, but in here, worker-1 is actually a string, so using back quote doesn't make sense to me as it's normally used elsewhere for keywords, and I would just use single or double quotes instead.

For keywords such as --nprocs, maybe we should just use a single back quote, as there's no text parsing and instead only CLI prints? Or is Sphinx parsing dask-cuda-worker CLI arguments now?

Same comments apply to other cases below, if we decide to change.

Reply:
Yeah I chose the backticks here as it's being formatted by sphinx-click now, the same way a docstring would be:

![image](https://user-images.githubusercontent.com/20627856/113726471-43c8ff80-96c2-11eb-9dec-86aa03858c64.png)

I agree that adding quotes to strings would help distinguish them from options, although I'm not sure the best way to address the backticks in the CLI output.

Reply:
Maybe then it makes sense to write ``"worker-1"`` ?

Double backticks for CLI output are ok then, given we now have Sphinx autogenerating formatted output.